source/opt/instruction.h

```diff
@@ -806,6 +810,14 @@ inline void Instruction::ForEachInOperand(
   });
 }
 
+inline bool Instruction::HasBranchWeights() const {
+  if (opcode_ == SpvOpBranchConditional && NumOperands() == 5) {
```

Comment:
Once this is in the cpp file, declare a constant in the anonymous namespace at the top of the file to avoid using the magic number 5.  Give the constant a suitable name to indicate that it represents the number of operands of a branch with weights.

Reply:
Done.